oss/ossSocket.cpp

```diff
@@ -14,6 +14,8 @@
 */
 
 #include "ossSocket.hpp"
+#include "ossQueue.hpp"
+#include "ossXLatch.hpp"
```

Comment:
only imported by the `cpp` class those hpp files will be detected and compiled by the cmake